src/iface/del.rs

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+
+use crate::handle::nl80211_execute;
+use crate::{
+    Nl80211Attr, Nl80211Command, Nl80211Error, Nl80211Handle, Nl80211Message,
+};
+use futures::TryStream;
+use netlink_packet_core::{NLM_F_ACK, NLM_F_REQUEST};
+use netlink_packet_generic::GenlMessage;
+
+/// Request for deleting an 802.11 interface.
+pub struct Nl80211InterfaceDeleteRequest {
+    handle: Nl80211Handle,
+    attributes: Vec<Nl80211Attr>,
+}
+
+impl Nl80211InterfaceDeleteRequest {
+    /// Create a new Delete request.
+    ///
+    /// See <https://github.com/torvalds/linux/blob/v6.17/include/uapi/linux/nl80211.h#L380> for
```

Comment:
Instead of linking to the linux source code, either use the builder pattern as suggested below or list possible attributes / link to possible attributes, which are already defined as types here.